packages/server/graphql/types/User.ts

```diff
@@ -404,6 +405,37 @@ const User = new GraphQLObjectType<any, GQLContext>({
         return source.overLimitCopy
       }
     },
+    similarReflectionGroups: {
+      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(RetroReflectionGroup))),
+      description:
+        'The reflection groups that are similar to the selected reflection in the Spotlight',
+      args: {
+        reflectionId: {
+          type: new GraphQLNonNull(GraphQLID),
+          description: 'The id of the selected reflection in the Spotlight'
+        },
+        searchQuery: {
+          type: new GraphQLNonNull(GraphQLString),
+          description: 'Only return reflection groups that match the search query'
+        }
+      },
+      resolve: async ({id: userId}, {reflectionId}, {dataLoader}) => {
+        const retroReflection = await dataLoader.get('retroReflections').load(reflectionId)
+        if (!retroReflection) return []
+        const {meetingId} = retroReflection
+        const meetingMemberId = toTeamMemberId(meetingId, userId)
+        const [viewerMeetingMember, reflectionGroups] = await Promise.all([
+          dataLoader.get('meetingMembers').load(meetingMemberId),
+          dataLoader.get('retroReflectionGroupsByMeetingId').load(meetingId)
+        ])
+        const {meetingId: viewerMeetingId} = viewerMeetingMember
```

Comment:
i probably have a bad type on this dataloader so it's not suggesting that it could be null.
`viewerMeetingMember` is going to be null if `meetingMemberId` isn't valid.
That means you need validate viewerMeetingMember before you destructure it. e.g. `if (!viewerMeetingMember return error)`